plugins/request-uri-manipulations.rst

```diff
@@ -0,0 +1,26 @@
+Request URI Manipulations
+=========================
+
+Request URI manipulations can be done thanks to several plugins:
+
+* ``AddHostPlugin``: Set host, scheme and port. Depending on configuration,
+the host is overwritten in every request or only set if not yet defined in the request.
```

Comment:
if you want to have multiline in a bullet list, you need to indent the second line by two spaces to align with the ` above.